assets/src/hooks/useDetour.ts

```diff
@@ -213,14 +213,18 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
     /**
      * Removes the last waypoint in {@link waypoints} if {@link canUndo} is `true`.
      */
-    undo: state === DetourState.Finished ? undefined : undo,
+    undo,
+    //   undo: state === DetourState.Finished ? undefined : undo,
     /**
      * Clears the entire detour
      */
-    clear: state === DetourState.Finished ? undefined : clear,
+    clear,
+    //   clear: state === DetourState.Finished ? undefined : clear,
+
     /** When present, puts this detour in "finished mode" */
     finishDetour: endPoint !== null ? finishDetour : undefined,
     /** When present, puts this detour in "edit mode" */
-    editDetour: state === DetourState.Finished ? editDetour : undefined,
+    editDetour,
```

Comment:
Dead code: `editDetour` is hidden behind a button that's only visible in `DetourState.Finished`.